### DISCUSSION TITLE
Can I do it with shortcut ?

### DISCUSSION THREAD

**Discussion by crnirg** [sumatrapdfreader/sumatrapdf#2868] [Q&A]
Shortcuts [
	[
		Cmd = CmdTranslateSelectionWithGoogle
		Key = t
	]

]

**Comment by username116** [comment]
Yes, it works for me.

(Another solution: I use [Qtranslate](https://quest-app.appspot.com/), it allows to translate the text in any window: the shortcut Ctrl+Q show translation in a popup window. I use it all the time)

**Reply by crnirg** [reply]
Hvala !

**Comment by crnirg** [comment]
The problem was because it was started as usual and not as adm.
